sympy/sets/fancysets.py

```diff
@@ -497,30 +497,36 @@ def normalize_theta_set(theta):
 
     if theta.is_Interval:
         # one complete circle
-        if Abs(theta.args[0] - theta.args[1]) >= 2*S.Pi:
+        intrvl_length = theta.measure
+        if intrvl_length >= 2*S.Pi:
+            if intrvl_length == 2*S.Pi and theta.left_open and theta.right_open:
+                return Interval(0, 2*S.Pi, True, True)
```

Comment:
The set of theta-values in the interval ``[0, 2*pi)`` congruent to those of ``(pi, 3*pi)`` modulo ``2*pi`` consists of all values but ``pi``. The subinterval ``[2*pi, 3*pi)`` should be shifted to ``[0, pi)``.